src/Directory.Build.props

```diff
@@ -17,6 +17,7 @@
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/DataDog/dd-trace-dotnet.git</RepositoryUrl>
     <Copyright>Datadog, Inc. 2017-2019</Copyright>
+    <Company>Datadog</Company>
```

Comment:
If we don't specify `<Company>`, the `[AssemblyCompanyAttribute]` will use the value from `<Authors>` instead.